Makefile

```diff
@@ -234,7 +265,7 @@ ifneq "${OSSM_MANIFEST_DATE}" ""
 endif
 
 .PHONY: update-charts
-update-charts: update-1.1-charts update-2.0-charts update-2.1-charts update-2.2-charts
+update-charts: update-1.1-charts update-2.0-charts update-2.1-charts update-2.2-charts update-2.3-charts
 
 .PHONY: update-templates
 update-templates: update-1.1-templates update-2.0-templates update-2.1-templates
```

Comment:
I think we need to add `update-2.2-templates` here.

Reply:
Done